### DISCUSSION TITLE
Claroline install 13.2.1

### DISCUSSION THREAD

**Discussion by kiral54** [claroline/Claroline#2060] [Q&A]
Hi,
Command "php bin/console claroline:install -vvv" was unable to build themes during 13.2.1 installation.
Nethertheless i tried to run application and apache server logically returned  a 500 Internal Server Error. All requirements are satisfied. 
The prod.log file says "request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Cannot find theme generated assets file(s). Make sure you have built them.")." at /Claroline/src/plugin/tag/Resources/views/layout/stylesheets.html.twig line 2 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Cannot find theme generated assets file(s). Make sure you have built them.\"). at /Claroline/src/plugin/tag/Resources/views/layout/stylesheets.html.twig:2, Exception(code: 0): Cannot find theme generated assets file(s). Make sure you have built them. at /Claroline/src/main/theme/Twig/ThemeExtension.php:89)"} []"

Furthermore, a second log error says "request.CRITICAL: Uncaught PHP Exception LightSaml\Error\LightSamlBuildException: "There are no own credentials" at /Claroline/vendor/lightsaml/lightsaml/src/LightSaml/Store/Credential/Factory/CredentialFactory.php line 78 {"exception":"[object] (LightSaml\\Error\\LightSamlBuildException(code: 0): There are no own credentials at /Claroline/vendor/lightsaml/lightsaml/src/LightSaml/Store/Credential/Factory/CredentialFactory.php:78)"} []"

Help would be appreciate.
Thanks in advance.

**Comment by LaurentGruber** [comment]
Hi,

For the second error : go to files/config/bundles.ini and check that the line Claroline\SamlBundle\ClarolineSamlBundle=false is not at true. It's probably this error that made a 500 by apache.

For the template error could you try php bin/console claroline:theme:build and see what's return ?

**Reply by kiral54** [reply]
Hi Laurent,

French? 
Thanks a lot for your answer. Saml error has been fixed after clearing cache. The file "theme-assets.json" was missing throwing the exception.
The login form is now displaying correctly but when i click on connection button, nothing's happening.
Kind regards.

**Reply by kiral54** [reply]
Here is a screenshot from Chrome.

![Claroline](https://user-images.githubusercontent.com/99762069/154350435-9d686faf-43f7-425d-b6ca-7fa19124a766.PNG)

**Reply by kiral54** [reply]
Command "composer require symfony/apache-pack" is supposed to create .htaccess file with rewrite rules. It didn't for me. 
Next step tomorrow. 
Good evening.

**Reply by LaurentGruber** [reply]
yes french :p.

for the error claro_security_login the answer is here : https://github.com/claroline/Claroline/discussions/1648#discussioncomment-588028

I don't use .htaccess file, so I can't help you for that, but I can find the apache vhost config for you if you want.

**Comment by kiral54** [comment]
Bonjour Laurent,

Merci pour ta réponse. 
J'avais effectivement vu la configuration pour le nom de domaine, l'url et ssl dans le fichier platform_options.json. Je l'avais donc changée mais l'erreur persistait malgré tout.
Du coup je suis preneur pour la config Apache.
Merci d'avance.
Cordialement.

**Reply by LaurentGruber** [reply]
La config de base pour un projet symfony doit suffire : https://symfony.com/doc/current/setup/web_server_configuration.html#using-mod-proxy-fcgi-with-apache-2-4

genre ça : 

```
<VirtualHost *:80>
    ServerName domain.tld
    ServerAlias www.domain.tld

    DocumentRoot /var/www/project/public
    <Directory /var/www/project/public>
        AllowOverride All
        Order Allow,Deny
        Allow from All
    </Directory>

    # uncomment the following lines if you install assets as symlinks
    # or run into problems when compiling LESS/Sass/CoffeeScript assets
    # <Directory /var/www/project>
    #     Options FollowSymlinks
    # </Directory>

    ErrorLog /var/log/apache2/project_error.log
    CustomLog /var/log/apache2/project_access.log combined
</VirtualHost>
```

Sinon tu peux venir en discuter sur gitter, ça sera peut être plus simple : https://gitter.im/claroline/Claroline

**Comment by kiral54** [comment]
Super. Merci 
En cas de besoin je te contacterai sur gitter.
